lib/desktop/cli.rb

```diff
@@ -187,6 +187,7 @@ def cli_syntax(command, args_str = nil)
       cli_syntax(c, '[TYPE]')
       c.summary = 'Start an interactive desktop session'
       c.action Commands, :start
+      c.slop.string '-n', '--name', "Give the desktop session a name so it can be more easily identified.", delimiter: nil, meta: 'NAME'
```

Comment:
After a more thorough reading of the slop documentation, I now realise that `delimiter` is only required for array flags not string flags.  It specifies how to split a string into multiple arguments.  Can you remove `delimiter: nil` please?